src/EncCnvtrs/EncCnvtrs 2010.csproj

```diff
@@ -146,7 +146,7 @@
     </Reference>
     <Reference Include="ECInterfaces, Version=4.0.0.0, Culture=neutral, PublicKeyToken=f1447bae1e63f485, processorArchitecture=x86">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\output\release\ECInterfaces.dll</HintPath>
+      <HintPath>..\..\output\$(Configuration)\ECInterfaces.dll</HintPath>

```

Comment:
This shouldn't be necessary, output paths for both release and debug will be checked automatically by msbuild. Also I'm not sure this is valid for the HintPath in VisualStudio (I'd have to check and see).

Reply:
IIRC, the reason for this was because VSCode was using referenced libraries to resolve symbols and determine intellisense. And this allowed it to do that for some code I was working with. 
Notice, that this isn't even the actual project file that mono would be using (which would presumably be `EncCnvtrs-mono.csproj`). 
I did not look into why VSCode was using this `EncCnvtrs 2010.csproj` instead of the `-mono.csproj` file. I assume it was looking at a `.sln` file, or maybe picking the first `.csproj` in the directory. 
Now that I have edited the code in question, I won't be benefiting from VSCode's intellisense etc unless I revisit that area, and so I could change this back if it's thought to be a problem.